### ISSUE TITLE
[Accessibility]: Zoom bar triggers a violation "The SVG element has no accessible name"

### ISSUE THREAD

**Issue by jadamski02** [carbon-design-system/carbon-charts#2018]
### Application/Team

IBM Decision Optimization

### Environment

Browser

### What happened? What did you expect to happen instead?

I’m using a LineChart with the zoom bar enabled, but the zoom bar is triggering an accessibility violation: "The SVG element has no accessible name".
Zoom bar configuration:
```
zoomBar: {
	updateRangeAxis: true,
	top: {
		enabled: true,
		type: "slider_view",
		initialZoomDomain: [1, lastObjectiveTime + 0.1 * lastObjectiveTime]
	}
},
```

For the chart itself, I've set aria-label via props:
```
accessibility: {
	svgAriaLabel: "Objectives Chart",
}
```
It seems that the zoom bar does not have an aria-label, and I haven’t been able to find a way to set one.
I’d really appreciate it if you could add a prop that allows setting it.

<img width="1436" height="568" alt="Image" src="https://github.com/user-attachments/assets/8c1005b1-9596-4c70-b621-8b1137f787c9" />

<img width="3402" height="1748" alt="Image" src="https://github.com/user-attachments/assets/882a5802-5284-4e01-b955-433f7df46618" />

### What WCAG 2.1 checkpoint does the issue violate?

1.1.1 – Non-text Content

### Version

"@carbon/charts": "1.23.13",
@carbon/charts-react": "1.23.13"

### Data & options used

```js

```

### Relevant log output

```js

```

### What priority level would this issue be in your opinion?

P1

**Comment by theiliad** [comment]
Pls update to the latest version. That element now has a `role=presentation` and should be ignored by screen readers

<img width="724" height="170" alt="Image" src="https://github.com/user-attachments/assets/b0964cf0-d879-4b87-a735-c5693074b353" />
